.woodpecker.yml

```
@@ -148,76 +148,22 @@ pipeline:
     when:
       event:
         - cron
-          #platform: linux/amd64
 
-  publish_release_docker_image_amd:
+  publish_release_docker_image:
     image: woodpeckerci/plugin-docker-buildx
     settings:
       repo: dessalines/lemmy
       dockerfile: docker/Dockerfile.multiarch
-      platforms: linux/amd64
+      platforms: linux/amd64,arm64
```

Shouldn't this be 
```suggestion
      platforms: linux/amd64,linux/arm64
```
according to [the docker documentation](https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/)?